src/routes.ts

```diff
@@ -14,6 +16,11 @@ router.delete("/logout", passport.authenticate('jwt', { session: false }), auth.
 // User
 router.get('/user', passport.authenticate('jwt', { session: false }), user.userInfo);
 
+// Problems
+router.get("/problem/:problemId", problem.problemInfo);
+router.get("/submission/:submissionId", submission.submissionInfo);
+router.get("/problem/:problemId/submissions", submission.allSubmissionInfo);
```

Comment:
These endpoints require authentication.

Reply:
I've just fixed that. Please have a look.